.github/mergeable.yml

```diff
@@ -0,0 +1,269 @@
+                     Error : {{{details.error}}}
+                     {{/if}}
+                     {{/each}}
+              {{/each}}"
+
+  # Remove label 'status: needs review' once PR is approved.
+  - when: 'pull_request_review.*'
+    name: 'Status check'
+    validate:
+      - do: payload
+        review:
+          state:
+            must_include:
+              regex: 'approved'
+    pass:
+      - do: labels
+        delete: 'status: needs review'
+      - do: checks
+        status: neutral
+
+  # Add label 'status: needs work' if reviewer requested changes.
+  - when: 'pull_request_review.*'
+    name: 'Status check'
+    filter:
+      - do: payload
+        review:
+          state:
+            must_include:
+              regex: 'changes_requested'
+    validate:
+      - do: label
+        must_include:
+          regex: 'status: needs review'
+    pass:
+      - do: labels
+        add: 'status: needs work'
+        delete: 'status: needs review'
+      - do: comment
+        payload:
+          body: 'As the reviewer has requested changes, we have added "status: needs work" and have removed "status: needs review" changes.'
+
+  # Let's add "maintainers" team as reviewer once, 'optional: review requested' label is added.
+  - when: pull_request.*
+    name: 'Request mainatainer for review'
+    filter:
+      - do: payload
+          review:
+            state:
+              must_include:
+                regex: 'review_requested'
+    validate:
+      - do: label
+        begins_with:
+          match: 'optional: review_requested'
+    pass:
+      - do: request_review
+        teams: ['maintainers']
+      - do: comment
+          payload:
+            body: 'As the @author has requested review from maintainers, we have added @goalgorilla/maintainers reviewers.'
+
+  # We want to make sure that the cherry-pick was completed after the PR was merged.
+  # This check will run after 24 hours of PR merging.
+  # see: https://mergeable.readthedocs.io/en/latest/recipes.html?highlight=stale#check-stale-pr-and-issues
+  - when: schedule.repository
+    name: 'Check backport'
+    type: pull_request
+    time_constraint:
+      time_zone: 'Europe/Amsterdamn'
+      hours_between: [ '9', '17' ]
+      days_of_week: [ 'Mon', 'Tue', 'Wed', 'Thu', 'Fri' ]
+    filter:
+      and:
+        - do: payload
+          pull_request:
+            merged:
+              boolean:
+                match: true
+        - do: payload
+          pull_request:
+            labels:
+              must_include:
+                regex: 'status: needs backport'
+                key: 'name'
+    validate:
+      and:
+        - do: stale
+          days: 1
+          type: pull_request
+        - do: label
+          must_include:
+            regex: 'verified-cherry-picks'
+            message: 'Cherry pick flag is required'
+    pass:
+      - do: comment
+        payload:
+          body: 'Great job! Please relax. The PR is complete.'
```

Comment:
Not sure if this is really true, I like the messaging in a way. But if there are still other open comments this might not be true, you might want to put this in a separate array only when all checks pass

Reply:
I have removed it.